app/assets/v2/js/grants/stats.js

```diff
@@ -0,0 +1,48 @@
+google.charts.load('current', { packages: [ 'corechart', 'bar' ]});
+google.charts.setOnLoadCallback(drawChart);
+
+function drawChart() {
+  var target = document.getElementById('grant_chart');
+
+  if (!target) {
+    return;
+  }
+  var data = google.visualization.arrayToDataTable(document.history);
+
+  var view = new google.visualization.DataView(data);
+  var width = parseInt($('#grant_stats_graph').width());
+
+  var ticks = [];
+  var increment = 100;
+
+  if (document.max_graph > 1000) {
+    increment = 1000;
+  }
+  if (document.max_graph > 10000) {
```

Comment:
else if

Reply:
thatll change the logical execution of this. i could move line 21 above line 18 + add an else if tho